packages/components/src/popover/index.js

```diff
@@ -55,6 +55,10 @@ class Popover extends Component {
 			isMobile: false,
 			popoverSize: null,
 		};
+
+		// Property used keep track of the previous anchor rect
+		// used to compute the popover position and size.
+		this.anchorRect = {};
```

Comment:
Can't we just say rectangle? We can spare the bytes 😉 

```suggestion
		this.anchorRectangle = {};
```

Reply:
I guess a bunch of the DOM APIs use `Rect`, so I suppose for consistency it's fine. I just dislike it, but I guess my beef is with JS, not this variable name 😉

Reply:
yes, since this is mainly based on `getBoundingClientRect` I'd be in favor of keeping "rect" at the moment.